src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray_1.cs

```diff
@@ -823,7 +823,6 @@ bool ICollection<T>.Remove(T item)
         /// <summary>
         /// See <see cref="IImmutableList{T}"/>
         /// </summary>
-        [ExcludeFromCodeCoverage]
```

Comment:
I removed the attributes outright in cases where I didn't see a good reason to exclude them, e.g. ImmutableArray's IImmutableList implementation should be accessible and tested.